static/panes/compiler.ts

```diff
@@ -1453,11 +1409,14 @@ export class Compiler extends MonacoPane<monaco.editor.IStandaloneCodeEditor, Co
             if (result.languageId) {
                 monaco.editor.setModelLanguage(editorModel, result.languageId);
             } else {
-                const monacoDisassembly =
-                    // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
-                    (languages[this.currentLangId ?? '']
-                        ? languages[this.currentLangId ?? ''].monacoDisassembly
-                        : null) || 'asm';
+                let monacoDisassembly = 'asm';
+                if (this.currentLangId && this.currentLangId in languages) {
```

Comment:
Probably not needed, compiler has to have a language. Only case it won't work is in dev if you open a state that has a compiler opened with a language that you disabled between reloads